Bottom: 4c316d73e1ea505379bb6dbbddebe6f151d026e7
Top:    8edc669fe335e3318dda295784af2b7e26fd908b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:08 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 1051207..25d2515 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -73,7 +73,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 #define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 47499ce..1d725f3 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 
 HID step_hid;
